### DISCUSSION TITLE
Guest View for Albums

### DISCUSSION THREAD

**Discussion by windmueller** [ente-io/ente#3278] [Enhancements]
I am using the new guest view quite often. However, currently it is only possible to select specific pictures for it, not whole albums. If an album only contains several days it is easy to mark the days one by one, but it would be quite helpful to have a "Guest view" action in the album menu.

**Comment by ua741** [comment]
Make sense.
>  If an album only contains several days it is easy to mark the days one by one
For the time being, You can select any photo, click on `All`, and then press `Guest view`.

**Comment by windmueller** [comment]
> For the time being, You can select any photo, click on All, and then press Guest view.

Interesting, I never noticed the "All" button before when selecting an image. That is a nice workaround, thank you!
